android/app/src/main/java/fi/fmi/mobileweather/BaseWidgetConfigurationActivity.java

```diff
@@ -4,6 +4,11 @@
 import static android.appwidget.AppWidgetManager.EXTRA_APPWIDGET_ID;
 import static android.appwidget.AppWidgetManager.INVALID_APPWIDGET_ID;
 
+import static fi.fmi.mobileweather.PrefKey.THEME;
+import static fi.fmi.mobileweather.Theme.DARK;
+import static fi.fmi.mobileweather.Theme.GRADIENT;
+import static fi.fmi.mobileweather.Theme.LIGHT;
```

Comment:
System default (automatic) is also one option and seems to be missing now

Reply:
I implement system default now with this ticket: https://github.com/fmidev/weather-app/issues/711